### ISSUE TITLE
Wrap guides are too thin under X11 with 1.5 scale factor

### ISSUE THREAD

**Issue by FreddieWitherden** [zed-industries/zed#19805]
### Check for existing issues

- [X] Completed

### Describe the bug / provide steps to reproduce it

In `settings.json` put:
```
  "wrap_guides": [80],
```
which under X11 with a 1.5 scale factor renders a line which is extremely thin (see attached screenshot).  Note, however, that the indentation lines on the left hand side are fine (1px for the wrap guides and 2px for the indentation lines).  This issue does not occur on macOS.

### Environment

Zed: v0.158.1 (Zed)
OS: Linux X11 gentoo 2.15
Memory: 62.4 GiB
Architecture: x86_64
GPU: Intel(R) Arc(tm) A770M Graphics (DG2) || Intel open-source Mesa driver || Mesa 24.2.5

### If applicable, add mockups / screenshots to help explain present your vision of the feature

![zed](https://github.com/user-attachments/assets/43fe4abf-60e2-446a-aef9-c12300ed1211)

### If applicable, attach your Zed.log file to this issue.

<details><summary>Zed.log</summary><pre>

</pre></details>

**Comment by andreyrd** [comment]
Can confirm on my end as well, but Wayland with 1x scale factor.
The line becomes a lot more visible if I set `soft_wrap` to `preferred_line_length`.
Otherwise, it's basically invisible.

**Comment by iamnbutler** [comment]
I don't think this is a design issue (it should just render the line correctly!) it may be an issue with gpui/a linux specific case or the blade render.

**Comment by FreddieWitherden** [comment]
Am able to reproduce as described with the current version.

**Comment by SomeoneToIgnore** [comment]
As a guess, https://github.com/zed-industries/zed/pull/38967 could help with this as helped with https://github.com/zed-industries/zed/issues/22711 

Zed Preview 0.207 (will be released later today) should contain the change — would be interesting to see if it really helps.

**Comment by FreddieWitherden** [comment]
Confirmed as still present in 0.207.

**Comment by FreddieWitherden** [comment]
Confirmed.

**Comment by yajo** [comment]
> I couldn't see the lines on "One Light" theme, and slightly on "One Dark".

I can see both now, the themes must have changed at some point.

**Comment by FreddieWitherden** [comment]
Confirmed still an issue on 1.10.3.
